iroh-net/src/magic_endpoint.rs

```diff
@@ -673,7 +672,8 @@ mod tests {
                     let mut buf = [0u8, 5];
                     stream.read_exact(&mut buf).await.unwrap();
                     info!("Accepted 1 stream, received {buf:?}.  Closing now.");
-                    ep.close(7u8.into(), b"bye").await.unwrap();
+                    // close the stream
+                    conn.close(7u8.into(), b"bye");
```

Comment:
by not awaiting this you're never closing this?

Reply:
there is no way to actually wait for a connection close asfaik